app/src/main/java/com/yacgroup/yacguide/SectorActivity.kt

```diff
@@ -83,6 +83,14 @@ class SectorActivity : UpdatableTableActivity() {
         }
     }
 
+    override fun showParent(v: View) {
```

Comment:
Please add a parent button to the RegionActivity, too.

Reply:
Right, missed that one.